app/views/layouts/_header.html.slim

```diff
@@ -52,6 +52,15 @@
                 = link_to "Swift", swift_learning_materials_path
               li
                 = link_to "VB.NET", vb_dot_net_learning_materials_path
+          li.dropdown.no-image
```

Comment:
does it look ok on the desktop? Space wise that is. We have already quite a few items in the header

Reply:
looks fine on mine. But yeah things can be better categorized and refactored. Let me try and do that.